specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2023-04-01-preview/loadtestservice.json

```diff
@@ -0,0 +1,3294 @@
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "TestRun"
+        ],
+        "summary": "List server metrics configuration for the given test run.",
+        "operationId": "LoadTestRun_GetServerMetricsConfig",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/TestRunIdPathParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Requested server metrics configuration",
+            "schema": {
+              "$ref": "#/definitions/TestRunServerMetricConfig"
+            }
+          },
+          "default": {
+            "description": "Load Testing service error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseBody"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List server metrics configuration for a test run": {
+            "$ref": "./examples/ListTestRunServerMetricsConfig.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AppComponent": {
+      "description": "An Azure resource object (Refer azure generic resource model : https://docs.microsoft.com/en-us/rest/api/resources/resources/get-by-id#genericresource)",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "fully qualified resource Id e.g subscriptions/{subId}/resourceGroups/{rg}/providers/Microsoft.LoadTestService/loadtests/{resName}",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceName": {
+          "description": "Azure resource name, required while creating the app component.",
+          "type": "string"
+        },
+        "resourceType": {
+          "description": "Azure resource type, required while creating the app component.",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Azure resource display name",
+          "type": "string"
+        },
+        "resourceGroup": {
+          "description": "Resource group name of the Azure resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "subscriptionId": {
+          "description": "Subscription Id of the Azure resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "kind": {
+          "description": "Kind of Azure resource type",
+          "type": "string"
+        }
+      }
+    },
+    "AutoStopCriteria": {
+      "description": "Auto stop criteria for a test.",
+      "type": "object",
+      "properties": {
+        "isAutoStopEnabled": {
```

Comment:
Our recently adopting naming guidelines recommend against using an "is" prefix for boolean fields. Would you be willing to make this "autoStopEnabled"?